aten/src/TH/generic/THBlas.cpp

```diff
@@ -380,6 +383,39 @@ void THBlas_(gemm)(
     return;
   }
 #endif
+
+#ifdef USE_FBGEMM
```

Comment:
Since the implementations of these functions will now in some cases use FBGEMM instead of blas/lapack, should we be changing the (file and/or function) names too?

Reply:
Thanks for the suggestion!

FBGEMM can be viewed as a low precision BLAS (with int8/int64 GEMM as well as other functionality like fp32/int8 SPMDM, etc. for the typical cases in the deep learning). As you can observe here the function signature for `fbgemm::cblas_gemm_i64_i64acc` is quite similar to `cblas_sgemm`.

I am open to change the file/function names. Let me know if you have any good suggestions :)

Reply:
I see the `blas` in the name of the function, so maybe it's fine as is. I'm not really clear on how fbgemm relates to blas, since I was thinking of blas as referring to a specific library. We could consider naming this in another PR if it does make sense.